hal/sparcv8leon3/gr716/gr716.c

```diff
@@ -317,6 +318,15 @@ int hal_platformctl(void *ptr)
 					&data->iocfg.dir, &data->iocfg.pullup, &data->iocfg.pulldn);
 			}
 			break;
+
+		case pctl_reboot:
+			if (data->action == pctl_set) {
```

Comment:
Perhaps a failsafe like on other targets?
```suggestion
			if ((data->action == pctl_set) && (data->reboot.magic == PCTL_REBOOT_MAGIC)) {
```